app/controllers/dish_photo_likes_controller.rb

```diff
@@ -5,6 +5,7 @@ class DishPhotoLikesController < ApplicationController
   def create
     if @dish_photo and !current_user.likes?(@dish_photo)
       current_user.like!(@dish_photo)
```

Comment:
`#like?` and `#like!` are methods defined in the [Socialization] (https://github.com/cmer/socialization) gem. Should I override it in `User` model?